app/code/core/Mage/Adminhtml/Block/Template.php

```diff
@@ -34,6 +34,13 @@
 class Mage_Adminhtml_Block_Template extends Mage_Core_Block_Template
 {
     /**
+     * Block event prefix
+     *
+     * @var string
+     */
+    $_eventPrefix = 'adminhtml';
```

Comment:
@LeeSaferite this event related edits shoulnt be part of this PR. So i closed it. But your right, should be changed.